sys/src/9/amd64/sdiahci.c

```diff
@@ -2002,7 +2002,8 @@ didtype(Pcidev *p)
 		    (p->did & 0xfffe) == 0x2922 ||	/* ich9 */
 		    p->did == 0x3a02 ||			/* 82801jd/do */
 		    (p->did & 0xfefe) == 0x3a22 ||	/* ich10, pch */
-		    (p->did & 0xfff8) == 0x3b28)	/* pchm */
+		    (p->did & 0xfff8) == 0x3b28 ||	/* pchm */
+		    p->did == 0xa102)			/* q170/q150/b150/h170/h110/z170/cm236 */
```

Comment:
```suggestion
		    p->did == 0xa102){ /* q170/q150/b150/h170/h110/z170/cm236 */
return Tich;
}

Reply:
This kind of thing should be fixed with clang-format maybe?